### PR TITLE
TextBlock: provide expected height

### PR DESCRIPTION
**Pull request by adrientetar**
r? @deltakosh

### CODE REVIEW

File: gui/src/2D/controls/textBlock.ts [hunk 1]
```diff
@@ -249,37 +269,62 @@ module BABYLON.GUI {
...
-            this.onLinesReadyObservable.notifyObservers(this);
+            return lines;
         }
 
         protected _parseLine(line: string = '', context: CanvasRenderingContext2D): object {
             return { text: line, width: context.measureText(line).width };
         }
 
-        protected _parseLineWithTextWrapping(line: string = '', context: CanvasRenderingContext2D): object {
+        protected _parseLineEllipsize(line: string = '', width: number,
```
Comment (deltakosh) [inline gui/src/2D/controls/textBlock.ts]: function name seems weird :)

Comment (adrientetar) [reply gui/src/2D/controls/textBlock.ts]: What do you suggest?

BTW not sure why CI fails, test suite passes for me locally (running on Firefox) and I did update the what's new file afaict...

Comment (deltakosh) [reply gui/src/2D/controls/textBlock.ts]: _parseLineEllipsis should work no?

Comment (deltakosh) [reply gui/src/2D/controls/textBlock.ts]: For the CI error: your are not updating the right what's new :)
You want to update the one in dist/preview release

Comment (adrientetar) [reply gui/src/2D/controls/textBlock.ts]: So the dist folder isn't all generated files? The what's new at the root of the repo is updated from the one in dist? [Seems things would be tidier w/o release files inside the repo tbh]

Comment (deltakosh) [reply gui/src/2D/controls/textBlock.ts]: Correct, not all files :)
The one at the root is for the current stable version.

Release files are in the repo to let people grab them directly

**Pull request merged**